src/Draco.Formatter.CSharp.Tests/FormatterTest.cs

```diff
@@ -0,0 +1,36 @@
+using Draco.Formatter.Csharp;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Text;
+using Xunit;
+using Xunit.Abstractions;
+namespace Draco.Formatter.CSharp.Test;
+public sealed class FormatterTest(ITestOutputHelper logger)
```

Comment:
```suggestion
using Xunit.Abstractions;

namespace Draco.Formatter.CSharp.Test;

public sealed class FormatterTest(ITestOutputHelper logger)
```

Reply:
Read the tests case more attentively :p.